### PR TITLE
[fix](stream_load)fix bug when stream without content-length or chunk… #27752

### PR DESCRIPTION
**Pull request by xingyingone**
1. forbid thed  stream_load without content-length or chunked Transfer Encoding
2. forbid thed  stream_load both with content-length and chunked Transfer Encoding

## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by xingyingone** [comment]
run   buildall

File: be/src/http/action/stream_load.cpp [hunk 1]
```diff
@@ -285,6 +285,19 @@ Status StreamLoadAction::_on_header(HttpRequest* http_req, std::shared_ptr<Strea
         }
     }
 
+    if (UNLIKELY((http_req->header(HttpHeaders::CONTENT_LENGTH).empty() &&
```
Comment (yiguolei) [inline be/src/http/action/stream_load.cpp:288]: Does master branch also has this problem?

File: be/src/http/action/stream_load.cpp [hunk 2]
```diff
@@ -285,6 +285,19 @@ Status StreamLoadAction::_on_header(HttpRequest* http_req, std::shared_ptr<Strea
...
+                  !ctx->is_chunked_transfer))) {
+        LOG(WARNING) << "content_length is empty and transfer-encoding!=chunked, please set "
+                        "content_length or transfer-encoding=chunked";
+        return Status::InternalError(
```
Comment (yiguolei) [inline be/src/http/action/stream_load.cpp:292]: DO NOT use internal error, because it is not an internal error. Maybe invalid input or invalid parameters?

**Comment by xiaokang** [comment]
#29169

File: be/src/http/action/stream_load.cpp (hunk 1 above)
Comment (xingyingone) [reply be/src/http/action/stream_load.cpp:288]: yes

**Comment by xingyingone** [comment]
run buildall

**Comment by xingyingone** [comment]
run buildall

**Comment by xingyingone** [comment]
run buildall

**Pull request merged**